lib/models/file-info.js

```diff
@@ -95,7 +97,33 @@ var FileInfo = CoreObject.extend({
     this._fileContents = fse.readFileSync(fullPath, { encoding: 'utf8' });
   },
 
+  updateImports: function() {
+    if (this.ext !== '.js' || !this._fileContents) { return; } // only process JavaScript files
+
+    var appName = this.options.projectName;
+
+    try {
+      var newContents = importDeclarationsTransform(
+        { source: this._fileContents,
+          fileInfo: this,
+          appName: appName,
+          fileInfos: this._fileInfoCollection._fileInfos
+        },
+        { jscodeshift: jscodeshift });
+
+      var fullPath = path.join(this.projectRoot, this.sourceRelativePath);
+
+      fse.writeFileSync(fullPath, newContents, { encoding: 'utf-8' });
+      this._fileContents = fse.readFileSync(fullPath, { encoding: 'utf8' });
```

Comment:
Do we need to re-read it?  Wouldn't `this._fileContents = newContents` be the same?